src/helm/config/model_deployments.yaml

```diff
@@ -167,7 +174,28 @@ model_deployments:
     client_spec:
       class_name: "helm.clients.bedrock_client.BedrockLlamaClient"
 
+  - name: amazon/llama3-1-405b-instruct-v1:0
+    model_name: meta/amazon-llama3-1-405b-instruct-v1:0
+    tokenizer_name: huggingface/gpt2
+    max_sequence_length: 2000
```

Comment:
Does the Bedrock version have a length of 2000  tokens? Most other versions of Llama 3.1 have a longer sequence length of 128K tokens.

Reply:
Discussed over email - 2000 tokens is the correct length as per the [Bedrock docs](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-meta.html).